src/test/java/org/springframework/integration/aws/kinesis/KclMessageDrivenChannelAdapterTests.java

```diff
@@ -116,6 +121,21 @@ void kclChannelAdapterReceivesRecords() {
 		assertThat(streamConsumers).hasSize(0);
 	}
 
+	@Test
+	public void metricsLevelOfMetricsFactoryShouldBeSetToMetricsLevelOfAdapter() {
+		MetricsLevel metricsLevel = TestUtils.getPropertyValue(
+			kclMessageDrivenChannelAdapter,
+			"scheduler.metricsFactory.metricsLevel",
+			MetricsLevel.class
+		);
+		MetricsLevel expectedMetricsLevel = TestUtils.getPropertyValue(
```

Comment:
We don't need to do this. There is just enough to compare with the `MetricsLevel.NONE`.